src/remote/remote.go

```diff
@@ -170,13 +170,14 @@ func (c *Client) initExec() error {
 	if err != nil {
 		return err
 	}
+	client.DefaultRPCTimeouts["default"] = time.Duration(c.state.Config.Remote.Timeout)
```

Comment:
I don't know if we can do something more sensible here. Or just lose the config option completely maybe?

Reply:
What's this used for? General timeouts for all RPC calls? I can't imagine many people setting this. The only reason I can think to have it configurable is if we need to increase/decrease based on the SLAs you've specced your environment for.